dist/server.js

```diff
@@ -35,12 +35,18 @@ var PARAMS = {
     style: { isRequired: true, isString: true },
     width: { isRequired: true, isInt: true },
     height: { isRequired: true, isInt: true },
-    zoom: { isRequired: false, isDecimal: true }
+    zoom: { isRequired: false, isDecimal: true },
+    ratio: { isRequired: false, isDecimal: true },
+    token: { isRequired: false, isString: true }
 };
 
 var renderImage = function renderImage(params, response, next, tilePath) {
     var width = params.width,
-        height = params.height;
+        height = params.height,
+        _params$token = params.token,
+        token = _params$token === undefined ? null : _params$token,
```

Comment:
@normanrz if I'm not mistaken, this is equivalent to `token = _params$token || null`, which avoids the ternary expression

Reply:
I think this comes from the babel transformation of `const { token = null } = params`. In this particular case it is quite unnecessary because we could just pass it as `null`/`undefined` to `render(...)`.